### PR TITLE
Add API 

### PR DESCRIPTION
**Pull request by newhorizon-tech**
- Use Leaderboard API to fetch the leaderboard
- Use Leaderboard API to post new scores on the leaderboard

### CODE REVIEW

File: src/api-interface.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const gameId = 'Rcl2sFmCQZv9b27vyvUs';
+
+const targetUrl = `${apiUrl}games/${gameId}/scores`;
```
Comment (o-rashad) [inline src/api-interface.js]: While you saved a leg work for you to define manually the game ID 🤓 
it's a requirement to create a game using the API and use its retrieved ID to submit the scores. [LINK](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Create%20a%20new%20game%20with%20the%20name%20of%20your%20choice%20by%20using%20the%20API.)

Kindly make a new fetch request in `async/await` style to create a new game with the help of the API documentation. 📜

**Review by o-rashad — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hi @newhorizon-tech 👋😄, I'm Omar and I will be your reviewer for this project 😊

Great job so far!👌😃
- The app is working as expected 👌🏽 
- You make good use of ES6 syntax and `async/await` funtions💯 
- The `README` file is nice and well organized. ⭐⭐

Only a final requirement to keep you moving forward : that you still need to work on to go to the next project but you are almost there! 🏁🏁🏁

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

You can also consider:
- [OPTINAL] Adding some validation to your name and score inputs to make sure nothing unexpected to pass.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api-interface.js (hunk 1 above)
Comment (newhorizon-tech) [reply src/api-interface.js]: Added a new fetch request

**Review by Dipeshtwis — APPROVED** [review]
# Hi @newhorizon-tech, 

### Status: Approved ✔️

:+1: You have implemented the suggested changes. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- In `package.json` the start command should look like this `npx webpack serve --open` then it will open the browser otherwise it's not opening currently.

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**